### PR TITLE
Added support for Sitecore 9.3.0 as well as Solr 8.1.1

### PR DESCRIPTION
**Pull request by klpatil**
Creating PR to support Sitecore 9.3.0 and Solr 8.1.1

config.yml comment is not updated to mention 9.3.0 - Somehow my Notepad++ was adding some character.

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-  sitecorePrefix: "demo"
+  accountName: "HorizontalIntegration"
+  deploymentUid: "ss300127"
+  sitecorePrefix: "sitecore"
   pathToWWWRoot: "C:\\inetpub\\wwwroot"
   solrUsername: ""
   solrPassword: ""
-  sitecoreVersion : "9.2.0" #Supported versions - 9.0.2, 9.1.1, and 9.2.0
\ No newline at end of file
+  sitecoreVersion : "9.3.0" #Supported versions - 9.0.2, 9.1.1, and 9.2.0
```
Comment (vtml) [inline config.yml:8]: Should update the example to have 9.3 in there

File: config.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 settings:
-  accountName: "MyAccountName"
-  deploymentUid: "ss881269"
-  sitecorePrefix: "demo"
+  accountName: "HorizontalIntegration"
+  deploymentUid: "ss300127"
+  sitecorePrefix: "sitecore"
```
Comment (vtml) [inline config.yml:4]: This should really be reverted back to the demo ones, as you don't want anyone knowing your account details

Comment (klpatil) [reply config.yml:4]: Thanks!

File: config.yml (hunk 1 above)
Comment (klpatil) [reply config.yml:8]: Sure

**Pull request closed without merging**